examples/Tutorial.ipynb

```diff
@@ -0,0 +1,925 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7d1dcb07",
+   "metadata": {},
+   "source": [
+    "# Piff Tutorial\n",
+    "\n",
+    "### Links\n",
+    "\n",
```

Comment:
As a general comment, I recently put together some recommendations to make Jupyter Notebooks more accessible. Can you access the following page?: https://confluence.lsstcorp.org/pages/viewpage.action?pageId=192909126 
If so, at the bottom there is a section called "Jupyter Notebooks resources for vision-impaired astronomers". I know it seems like a lot and I'd be happy to discuss it further with you if you want :)  There are recommendations such as: " Use colorblind intuitive palettes for graphics. As pointed out above, we may already be doing this in some tutorials (e.g., NB03), but we should check them all. 
Maximize accessibility for people with dyslexia. Use dyslexic-accessible fonts (including but not limited to Helvetica, Verdana, and OpenDyslexic) at a reasonably large font size (12 pt = 16 px) with sufficient space between lines. Avoid crowding, and follow best practices related to the length and arrangement of 2 sentences that are displayed. Many potential astronomy students have dyslexia 3 (Schneps 2006), and some people with disabilities are excluded because they feel incompetent when the situation is mostly related to access to information."

Reply:
I think everything is already pretty high contrast, and I think the font is already Helvetica or something similar.  (Also I don't actually know how to change it.)  Are there any specific changes you'd recommend related to this?